src/pagopa_proxy.tf

```diff
@@ -0,0 +1,146 @@
+resource "azurerm_resource_group" "proxy_rg" {
+  count    = var.proxy_enabled ? 1 : 0
+  name     = format("%s-proxy-rg", local.project)
+  location = var.location
+
+  tags = var.tags
+}
+
+module "proxy_snet" {
+  count                = var.proxy_enabled && var.cidr_subnet_proxy != null ? 1 : 0
+  source               = "git::https://github.com/pagopa/azurerm.git//subnet?ref=v1.0.51"
+  name                 = format("%s-proxy-snet", local.project)
+  address_prefixes     = var.cidr_subnet_proxy
+  resource_group_name  = azurerm_resource_group.rg_vnet.name
+  virtual_network_name = module.vnet.name
+
+  enforce_private_link_endpoint_network_policies = true
+
+  delegation = {
+    name = "default"
+    service_delegation = {
+      name    = "Microsoft.Web/serverFarms"
+      actions = ["Microsoft.Network/virtualNetworks/subnets/action"]
+    }
+  }
+}
+
+module "proxy_redis" {
+  count                 = var.proxy_enabled ? 1 : 0
+  source                = "git::https://github.com/pagopa/azurerm.git//redis_cache?ref=v1.0.37"
+  name                  = format("%s-proxy-redis", local.project)
+  resource_group_name   = azurerm_resource_group.proxy_rg[0].name
+  location              = azurerm_resource_group.proxy_rg[0].location
+  capacity              = var.proxy_redis_capacity
+  enable_non_ssl_port   = true
+  family                = var.proxy_redis_family
+  sku_name              = var.proxy_redis_sku_name
+  enable_authentication = true
+  subnet_id             = length(module.proxy_snet[0].*.id) == 0 ? null : module.proxy_snet[0].id
+
+  tags = var.tags
+}
+
+module "proxy_app_service" {
+  count  = var.proxy_enabled ? 1 : 0
+  source = "git::https://github.com/pagopa/azurerm.git//app_service?ref=v1.0.14"
+
+  resource_group_name = azurerm_resource_group.proxy_rg[0].name
+  location            = var.location
+
+  # App service plan vars
+  plan_name     = format("%s-plan-proxy", local.project)
+  plan_kind     = "Linux"
+  plan_sku_tier = var.proxy_tier
+  plan_sku_size = var.proxy_size
+  plan_reserved = true # Mandatory for Linux plan
+
+  # App service plan
+  name                = format("%s-app-proxy", local.project)
+  client_cert_enabled = false
+  always_on           = var.proxy_always_on
+  linux_fx_version    = "NODE|14-lts"
+  health_check_path   = "/ping"
+
+  app_settings = {
```

Comment:
To review app settings according to the current configuration for IO subscription --> https://github.com/pagopa/io-infrastructure-live-new/blob/43d89ce34ecef945dadefdb1dc6dc5473a1bd20a/prod/westeurope/external/pagopaproxyprod/app_service/terragrunt.hcl#L70